examples/basic-click-counter/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>React click conter example</title>
```

Comment:
Spelling: "conter" -> "counter"

And maybe change it to "Basic Example with Click Counter" to match the others.